### PR TITLE
[FIX] Issue #391

### PR DESCRIPTION
**Pull request by etobella**
Fixes issue #391 
It should only check internal locations.
@fanha99 can you check if this solves your problem?

### CODE REVIEW

**Comment by fanha99** [comment]
hi,
tested on runbot. 
FIXED.

File: stock_no_negative/models/stock_quant.py [hunk 1]
```diff
@@ -22,10 +22,13 @@ def check_negative_qty(self):
...
-            if (float_compare(quant.quantity, 0, precision_digits=p) == -1 and
-                    quant.product_id.type == 'product' and
-                    not quant.product_id.allow_negative_stock and
-                    not quant.product_id.categ_id.allow_negative_stock):
+            if (
+                float_compare(quant.quantity, 0, precision_digits=p) == -1 and
+                quant.product_id.type == 'product' and
+                not quant.product_id.allow_negative_stock and
+                not quant.product_id.categ_id.allow_negative_stock and
+                quant.location_id.usage in ['internal']
```
Comment (JordiBForgeFlow) [inline stock_no_negative/models/stock_quant.py]: I would add also 'transit'

Comment (etobella) [reply stock_no_negative/models/stock_quant.py]: Added

**Review by fanha99 — APPROVED** [review]

**Review by rgarnau — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please change module version number

**Comment by etobella** [comment]
@pedrobaeza Changed version

**Comment by pedrobaeza** [comment]
Travis is now red

**Comment by etobella** [comment]
Now it's green. It had some problem on  job because it could not download all the projects. I rexecuted it and it worked.

**Pull request merged**